source/src/retrosynthetic-planning/utility/BruteForceSearch.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import copy
+
+# reactions_dictionary = np.load('D:\工作\论文\分子逆合成/retrosynthesis\data2/reactions_dictionary.npy', allow_pickle=True).item()
+# target_product = np.load('D:\工作\论文\分子逆合成/retrosynthesis\data2/target_product.npy').tolist()
+# Deadend = np.load('D:\工作\论文\分子逆合成/retrosynthesis\data2/deadend.npy').tolist()
+# buyable = np.load('D:\工作\论文\分子逆合成/retrosynthesis\data2/buyable.npy').tolist()
```

Comment:
Please change Chinese description to English description, the same for the following files